frontend/packages/ux-editor/src/utils/datamodel.ts

```diff
@@ -47,6 +47,16 @@ export const getMinOccursFromDataModel = (
   return element?.minOccurs;
 };
 
+export const getMaxOccursForGroupFromDataModel = (
```

Comment:
The function itself does not need to know that it is used in a group context 😊 
```suggestion
export const getMaxOccursFromDataModel = (
```